io/plugins/eu.esdihumboldt.hale.io.csv.ui/src/eu/esdihumboldt/hale/io/csv/ui/InstanceExportConfigurationPage.java

```diff
@@ -122,23 +119,17 @@ public void widgetSelected(SelectionEvent e) {
 				}
 			});
 
-			Label separatorLabel = new Label(page, SWT.NONE);
-			separatorLabel.setText("Warning! Feature types with no data are not selectable");
+			Table table = new Table(chooseFeatureTypes,
+					SWT.CHECK | SWT.MULTI | SWT.V_SCROLL | SWT.H_SCROLL);
 
-			// Set the text colour of the label to yellow
-			Color greyLabel = PlatformUI.getWorkbench().getDisplay()
-					.getSystemColor(SWT.COLOR_DARK_GRAY);
-			separatorLabel.setForeground(greyLabel);
+			featureTypeTable = new CheckboxTableViewer(table);
 
-			table = new Table(chooseFeatureTypes, SWT.CHECK | SWT.MULTI | SWT.SCROLL_PAGE);
-			table.setHeaderVisible(false);
-			table.setLinesVisible(false);
-			table.setBackground(PlatformUI.getWorkbench().getDisplay()
-					.getSystemColor(SWT.COLOR_LIST_BACKGROUND));
```

Comment:
Any specific reason to remove these other settings?

Reply:
These lines were removed because they were either redundant or unnecessary given the context:
1. Header visibility: `table.setHeaderVisible(false);`
By default, the table header is not visible unless explicitly set to true. Since the intention was to keep the header invisible, this line was redundant.
2. Lines visibility: `table.setLinesVisible(false);`
Similar to the header visibility, the default setting for table lines visibility is false unless set to true. This line was therefore redundant as well.
3. Background color: `table.setBackground(PlatformUI.getWorkbench().getDisplay().getSystemColor(SWT.COLOR_LIST_BACKGROUND));`
The background color setting was removed because the default background color is typically suitable for the UI design, and it wasn't necessary to explicitly set it in this case.